pkg/nrtupdater/nrtupdater.go

```diff
@@ -176,6 +178,24 @@ func (te *NRTUpdater) updateNRTInfo(nrt *v1alpha2.NodeResourceTopology, info Mon
 // Check nrt.OwnerReferences for Node references and update it so it has only one Node reference,
 // the one to the Node with the same name as the NRT.
 func (te *NRTUpdater) updateOwnerReferences(nrt *v1alpha2.NodeResourceTopology) {
+	addDaemonSetOwner(nrt)
```

Comment:
I think we should add the daemonset owner
1. only if explicitly enabled
2. only if the node ownership is explicitly disabled

the most correct thing to do is the node ownership. From a scheduler perspective, this (using the node ownership as main one) enables the transparent replacement RTE <=> NFD.